definitions/environments/node/flow_v0.155.x-/node.js

```diff
@@ -4,6 +4,14 @@ declare module 'fs/promises' {
   declare module.exports: $Exports<'fs'>['promises'];
 }
 
+declare module 'process' {
+  declare module.exports: typeof process;
```

Comment:
Maybe try using the type directly? 

```suggestion
  declare module.exports: Process;
```